components/templates/src/global_fns/mod.rs

```diff
@@ -516,6 +541,10 @@ mod tests {
     const TRANS_CONFIG: &str = r#"
 base_url = "https://remplace-par-ton-url.fr"
 default_language = "fr"
+languages = [
+    { code = "fr" },
```

Comment:
Oups, this line will prevent the tests added in #981 to pass...

Reply:
indeed